File: src/main/java/com/multicampus/gamesungcoding/a11ymarketserver/feature/seller/dto/SellerOrderSummaryResponse.java
package com.multicampus.gamesungcoding.a11ymarketserver.feature.seller.dto;

public record SellerOrderSummaryResponse(long newOrders,
                                         long acceptedOrders,
                                         long shippingOrders,
                                         long completedOrders,
                                         long claimedOrders) {
}

File: src/main/kotlin/com/multicampus/gamesungcoding/a11ymarketserver/feature/seller/dto/SellerOrderSummaryResponse.kt
package com.multicampus.gamesungcoding.a11ymarketserver.feature.seller.dto

data class SellerOrderSummaryResponse(
    val newOrders: Long,
    val acceptedOrders: Long,
    val shippingOrders: Long,
    val completedOrders: Long,
    val claimedOrders: Long
)
